src/ucp/core/ucp_ep.c

```diff
@@ -1953,3 +1998,17 @@ uint64_t ucp_ep_get_tl_bitmap(ucp_ep_h ep)
 
     return tl_bitmap;
 }
+
+void ucp_ep_invoke_err_cb(ucp_ep_h ep, ucs_status_t status)
+{
+    if ((ucp_ep_config(ep)->key.err_mode == UCP_ERR_HANDLING_MODE_NONE) ||
```

Comment:
please document the three cases that are covered here for which the err_cb should not be invoked.